orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/handler/V1TemplateLoaderHandler.kt

```diff
@@ -37,6 +37,7 @@ class V1TemplateLoaderHandler(
 
   override fun handle(chain: HandlerChain, context: PipelineTemplateContext) {
     val config = objectMapper.convertValue(context.getRequest().config, TemplateConfiguration::class.java)
+    config.configuration.notifications.addAll(context.getRequest().notifications)
```

Comment:
I don't think we need to dedupe notifications, but let me know if you disagree

Reply:
We probably do, actually. I'm not confident that orca / echo would know enough to dedupe further down.

Reply:
ok. I am still not clear how `config.configuration.notification` would get filled since `deck` doesn't do that, but obviously possible via API

Reply:
I guess the other comment is that one can create multiple notifications today that go to the same spot - they aren't deduped... But if you feel strongly I will add the dedupe logic here

Reply:
resolved offline: leave as is for now